spark/core/build.gradle

```diff
@@ -130,7 +130,8 @@ sparkVariants {
 
 // deal with the messy conflicts out there
 // Ignore the scalaCompilerPlugin configurations since it is immediately resolved to configure the scala compiler tasks
-configurations.matching{ it.name.contains('CompilerPlugin') == false }.all { Configuration conf ->
+configurations.matching{ it.name.contains('CompilerPlugin') == false && (it.name.contains("spark30") || it.name.contains("Spark30")) == false}.all { Configuration conf ->
+    println conf.name
```

Comment:
Did you mean to leave this in?

Reply:
Will delete this line with extreme prejudice